### ISSUE TITLE
Some questions 

### ISSUE THREAD

**Issue by Chengwei94** [PeterZZQ/scDART#3]
Hi Peter,

I am trying to do a benchmark on these methods. Since that there are quite a lot of parameters to choose from. Is there a guideline on how I should choose these parameters.

**Comment by PeterZZQ** [comment]
Most of the parameters here should be kept the same, including `seed`, `learning_rate`, `n_epochs `, `use_potential `, and `reg_d`.  Usually, you don't need to change `ts` too. The main parameter that you need to tune around with should be `latent_dim`, `reg_g`, and `reg_mmd`. `latent_dim` controls the number of latent dimensions, it increases with the increase of the complexity of the dataset. `reg_g` and `reg_mmd` control the regularization weight of gene activity model and MMD loss.

You can start with the default parameter in the `demo.ipynb`, which usually produces a stable output.  You can refer to the manuscript for the purpose of different parameters. `use_anchor` is set to True if you have anchor cells between two modalities (e.g. root cells of trajectory), it helps produce a more stable result. If you don't have the anchor cells, you can set it to False.

<img width="270" alt="image" src="https://user-images.githubusercontent.com/37580703/180118623-ae0f6786-ce60-4711-a9c3-6a50b52acfae.png">
